src/System.Data.SqlClient/src/System/Data/SqlClient/SNI/SNIPacket.cs

```diff
@@ -58,15 +50,11 @@ public string Description
         /// <summary>
         /// Packet validity
         /// </summary>
-        public bool IsInvalid => (_data == null);
+        public bool IsInvalid => _data is null;
 
-        /// <summary>
-        /// Packet data
-        /// </summary>
-        public void Dispose()
-        {
-            Release();
-        }
+        public bool MuxHeaderReserved => ((_flags & SNIPacketFlags.MuxHeaderReserved) == SNIPacketFlags.MuxHeaderReserved);
+
+        public bool MuxHeaderWritten => ((_flags & SNIPacketFlags.MuxHeaderWritten) == SNIPacketFlags.MuxHeaderWritten);
```

Comment:
I dont see this Property being read anywhere. Why is this needed?

Reply:
Both removed. There's a new ReservedHeaderSize field now.